examples/otlptrace/src/main/java/com/google/cloud/opentelemetry/example/otlptrace/OTLPTraceExample.java

```diff
@@ -33,22 +30,9 @@ public class OTLPTraceExample {
 
   private static OpenTelemetrySdk openTelemetrySdk;
 
-  private static OpenTelemetrySdk setupTraceExporter() throws IOException {
-    GoogleCredentials credentials = GoogleCredentials.getApplicationDefault();
-
-    // Register the TraceExporter with OpenTelemetry
-    return OpenTelemetrySdk.builder()
-        .setTracerProvider(
-            SdkTracerProvider.builder()
-                .addSpanProcessor(
-                    BatchSpanProcessor.builder(
-                            OtlpHttpSpanExporter.builder()
-                                .addHeader(
-                                    "Authorization", "Bearer " + credentials.getAccessToken())
-                                .build())
-                        .build())
-                .build())
-        .buildAndRegisterGlobal();
```

Comment:
we need to send the `Authorization` header in this use case to authenticate against GCP. Does including this override the autoconfig for endpoint?

Reply:
to clarify: we want to get `Authorization` programmatically (ie, not through env var), but we do want to set endpoint through an env var

Reply:
I addressed this in the latest commit.